src/Platform.Handlers/src/Xamarin.Platform.Handlers/Xamarin.Platform.Handlers.csproj

```diff
@@ -1,8 +1,10 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
   <PropertyGroup>
-    <TargetFrameworks>$(TargetFrameworks);netstandard2.0;Xamarin.iOS10;MonoAndroid10.0;Xamarin.Mac20</TargetFrameworks>
+    <TargetFrameworks>netstandard2.1;Xamarin.iOS10;MonoAndroid10.0;Xamarin.Mac20</TargetFrameworks>
     <RootNamespace>Xamarin.Platform</RootNamespace>
     <AssemblyName>Xamarin.Platform</AssemblyName>
+    <LangVersion>8.0</LangVersion>
+    <Nullable>enable</Nullable>
```

Comment:
This is the important change!

Reply:
I'd go one step further, enable it in Directory.Build.props, and disable it on projects (almost all) that aren't ready to support it yet. Same effect, different message

Reply:
Nice suggestion but currently can be only enable in the handlers project and 8.0 is already all setup everywhere.